.github/workflows/main.yml

```diff
@@ -14,19 +14,23 @@ jobs:
       - name: Checkout repo
         uses: actions/checkout@v2
 
+      - name: Set up PNPM
+        uses: pnpm/action-setup@c3b53f6a16e57305370b4ae5a540c2077a1d50dd #v2.2.4
```

Comment:
I don't mind if you use the version number here and we update all the actions all at once